hbs-helpers.js

```diff
@@ -16,7 +23,7 @@ function breachDataClasses(dataClasses) {
 
 function prettyDate(date) {
   const jsDate = new Date(date);
-  // TODO: localize this
+  // TODO: l10n
   return jsDate.toLocaleDateString("en-US", {year: "numeric", month: "long", day: "numeric"});
```

Comment:
Fluent can format dates, too. You can pass the date object as an argument to `format`. I assume you'd need to make `prettyDate` take `req` as an argument, however. I'm not sure if that's possible given where it's used.

```js
function prettyDate(req, date) {
  return fluentFormat(req, "pretty-date", {date: new Date(date)});
}
```
```properties
# en-US
pretty-date = {DATETIME($date, year: "numeric", month: "long", day: "numeric")}
```
```properties
# Some other language might decide long month names are too long.
pretty-date = {DATETIME($date, year: "numeric", month: "short", day: "numeric")}
```

There's also a not-very-well-documented feature of setting the default formatting options for the arguments.

```js
import {FluentDateTime} from "fluent";

function prettyDate(req, date) {
  return fluentFormat(req, "pretty-date", {
    date: new FluentDateTime(date, {
      year: "numeric",
      month: "long",
      day: "numeric",
    })
  });
}
```
```properties
# en-US
pretty-date = {$date}
```

Other locales can still customize the formatting options, if needed:

```properties
# Some other language might decide long month names are too long.
pretty-date = {DATETIME($date, year: "numeric", month: "short", day: "numeric")}
```